free-programming-books.md

```diff
@@ -680,6 +690,7 @@ Kerridge (PDF) (email address *requested*, not required)
 
 * [A Graduate Course in Applied Cryptography](http://toc.cryptobook.us)
 * [Crypto 101 - Crypto for everyone](https://www.crypto101.io)
+* [Cryptography & Security - Lecture notes](http://cse.unl.edu/~cbourke/cryptoNotes.pdf) - Dr. Chris Bourke (PDF)
```

Comment:
138 pages is short and dense with all Math support  but autonomous as the other "Lectures"  IMHO is worthy material (and above all open) 
Maybe in list we can open  a category say Monographs and put them there?

Reply:
No, it's not a monograph, either. It has more bullet lists than complete sentences.

Reply:
mmh so what we do then with Lectures? throw the m out? seems like a sin 
or add a new free-programming-lectures.md and put them there? 

PS: I used the term monograph loosely i haven't librarian knowledge some link-hint to take knowledge on that stuff?

Reply:
Encourage the prof to post the lectures on youtube so that together with the notes, we could put it in free-programming-courses. "Leaving things out" is another way to say for 'editing" != sin.

Reply:
> Encourage the prof to post the lectures on youtube so that together with the notes, we could put it in free-programming-courses. 

Nah i don't guess he uses multimedia(can't confirm because my old browser freeze on YT) and actually i haven't time/will for that stuff
In case weren't  easier create a new free-programming-lectures.md file and put Lectures material  there?    

> "Leaving things out" is another way to say for 'editing" != sin.

don't get that : have to get rid lectures?

Reply:
yes please.

Reply:
and ... so be it :) ; conv resolved